tests/testpm.c

```diff
@@ -21,7 +21,7 @@ int main(int argc, char * argv[]) {
         .nc = 32,
         .boxsize = 1600.,
         .alloc_factor = 2.0,
-        .omega_m = 0.292,
+        .cosmology = NULL,
```

Comment:
This uses the default cosmology branch. What about adding a fastpm_get_default cosmology function, and use that?